js/foundation.offcanvas.js

```diff
@@ -146,6 +147,16 @@ class OffCanvas {
   }
 
   /**
+   * Stops scrolling of the body when offcanvas is open on mobile Safari and other troublesome browsers.
+   * @private
+   */
+  _stopScrolling(event) {
+  	event.preventDefault();
+  	event.stopPropagation();
+  	return false;
```

Comment:
We believe return false is sufficient:  http://stackoverflow.com/questions/1357118/event-preventdefault-vs-return-false